<launch>
	
	<arg name="dimension" default="3"/>
	<arg name="nonlinear_opt" default="true"/>
	<arg name="input_middle" default="false"/>
	<arg name="max_v" default="0.8"/>
    <arg name="max_a" default="0.7"/>
	<arg name="max_ang_v" default="0.5"/>
    <arg name="max_ang_a" default="0.5"/>

	<node pkg="traj_gen" name="traj_gen_node" type="traj_gen_node" output="screen">
		<!-- <remap from="path_segments_4D" to="trajectory"/> -->
		<param name="dimension" value="$(arg dimension)"/>
		<param name="nonlinear_opt" value="$(arg nonlinear_opt)"/>
		<param name="input_middle" value="$(arg input_middle)"/>
		<param name="max_v" value="$(arg max_v)"/>
		<param name="max_a" value="$(arg max_a)"/>
		<param name="max_ang_v" value="$(arg max_ang_v)"/>
		<param name="max_ang_a" value="$(arg max_ang_a)"/>
		<rosparam param="target_pos">[0.0, 0.0, 8.0]</rosparam>
		<rosparam param="middle_pos">[2.0, 2.0, 3.0]</rosparam>
		<rosparam param="target_vel">[0.0, 0.0, 0.0]</rosparam>
	</node>

</launch>
